android/filamat-android/src/main/java/com/google/android/filament/filamat/MaterialBuilder.java

```diff
@@ -259,7 +259,7 @@ public MaterialBuilder uniformParameterArray(@NonNull UniformType type, int size
 
     @NonNull
     public MaterialBuilder samplerParameter(@NonNull SamplerType type, SamplerFormat format,
```

Comment:
You should add new methods for non-sampler parameters to specify the precision. Java doesn't have default parameter values, so just use overloading instead.

Reply:
Good catch, added a commit.